test/morphology/test_gradient.py

```diff
@@ -23,46 +21,46 @@ def test_cardinality(self, device, dtype, shape, kernel):
         assert gradient(img, krnl).shape == shape
 
     def test_value(self, device, dtype):
-        input = torch.tensor([[0.5, 1., 0.3], [0.7, 0.3, 0.8], [0.4, 0.9, 0.2]],
-                             device=device, dtype=dtype)[None, None, :, :]
-        kernel = torch.tensor([[0., 1., 0.], [1., 1., 1.], [0., 1., 0.]], device=device, dtype=dtype)
+        tensor = torch.tensor([[0.5, 1., 0.3], [0.7, 0.3, 0.8], [0.4, 0.9, 0.2]],
+                              device=device, dtype=dtype)[None, None, :, :]
+        kernel = torch.tensor([[-1., 0., -1.], [0., 0., 0.], [-1., 0., -1.]], device=device, dtype=dtype)
         expected = torch.tensor([[0.5, 0.7, 0.7], [0.4, 0.7, 0.6], [0.5, 0.7, 0.7]],
                                 device=device, dtype=dtype)[None, None, :, :]
-        assert_allclose(gradient(input, kernel), expected)
+        assert_allclose(gradient(tensor, kernel), expected)
 
     def test_exception(self, device, dtype):
-        input = torch.ones(1, 1, 3, 4, device=device, dtype=dtype)
+        tensor = torch.ones(1, 1, 3, 4, device=device, dtype=dtype)
```

Comment:
No problem with any name, but why change the code without a change? :)

Reply:
I changed the name because 'input' is a built-in function Python. However, it has no effect in this context. If you want I can put 'input' back :)

Reply:
no need to change it back again, that was just a comment for the future

Reply:
Thanks. I will keep it in mind :)